src/map/status.cpp

```diff
@@ -1925,10 +1925,13 @@ bool status_check_skilluse(struct block_list *src, struct block_list *target, ui
 	status_change *sc = NULL, *tsc;
```

Comment:
```suggestion
	status_change *sc, *tsc;
```

Reply:
Additionally: Can you move the tsc declaration to the respective line to prevent a similar issue with that variable please?